.jenkins/job_beam_PreCommit_Java_MavenInstall.groovy

```diff
@@ -35,7 +35,7 @@ mavenJob('beam_PreCommit_Java_MavenInstall') {
   common_job_properties.setMavenConfig(delegate)
 
   // Sets that this is a PreCommit job.
-  common_job_properties.setPreCommit(delegate, 'Jenkins: Maven clean install')
+  common_job_properties.setPreCommit(delegate, ['master'], 'Jenkins: Maven clean install')
```

Comment:
pull ['master'] out into its own var, called something like 'build_branches' and commented. Investigate whether the standard precommit should run on any branch other than master as well.